pkg/controllers/localbuild/nginx.go

```diff
@@ -3,133 +3,38 @@ package localbuild
 import (
 	"context"
 	"embed"
-	"errors"
-	"time"
 
 	"github.com/cnoe-io/idpbuilder/api/v1alpha1"
-	"github.com/cnoe-io/idpbuilder/pkg/k8s"
-	"github.com/cnoe-io/idpbuilder/pkg/util"
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
-	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 const (
-	nginxNamespace  string = "ingress-nginx"
-	nginxDeployment string = "ingress-nginx-controller"
+	nginxNamespace string = "ingress-nginx"
 )
 
 //go:embed resources/nginx/k8s/*
 var installNginxFS embed.FS
-var timeout = time.After(3 * time.Minute)
-
-func RawNginxInstallResources() ([][]byte, error) {
```

Comment:
Can we keep the ability to get embedded resources outside of these packages? Keep the functions or export the variable. The git repository reconciler needs a way to retrieve these to sync with gittea repos. You don't have to pass local dir ref and we have a distinction between repositories created with embedded contents vs local directory. 

```yaml
apiVersion: idpbuilder.cnoe.io/v1alpha1
kind: GitRepository
metadata:
  name: argocd
  namespace: default
spec:
  giteaURL: "http://localhost:3000"
  source:
    embeddedAppName: "argocd"
    type: embedded
```

Reply:
lets revisit it as part of pushing the files to the repos in the next PR. We will make adjustments where needed.